mne/io/brainvision/brainvision.py

```diff
@@ -498,11 +498,16 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale):
     for chan, props in cfg.items('Channel Infos'):
         n = int(re.findall(r'ch(\d+)', chan)[0]) - 1
         props = props.split(',')
+
         # default to microvolts because that's what the older brainvision
         # standard explicitly assumed; the unit is only allowed to be
         # something else if explicitly stated (cf. EEGLAB export below)
         if len(props) < 4:
-            props += (u'µV',)
+            props += ('µV',)
+        elif len(props) == 4 and props[-1] == '':
```

Comment:
Is `len(props) == 4` really necessary here? That is, are there cases where `len(props) > 4`? If not, I'd remove this condition and just use `props[-1] == ''` (or maybe even `props[3] == ''` to make it more explicit that we refer to the fourth entry (as opposed to the last).

Reply:
Good points, I've switched to `props[3]` now, this will also make things more future-proof in case the specs add new props someday!